platform/qt/qt.cmake

```diff
@@ -236,24 +279,25 @@ if(MSVC)
         set_target_properties(mbgl-core PROPERTIES COMPILE_PDB_NAME mbgl-cored)
         set_target_properties(qmapboxgl PROPERTIES COMPILE_PDB_NAME QMapboxGLd)
 
-        install(FILES $<TARGET_FILE_DIR:mbgl-vendor-csscolorparser>/mbgl-vendor-csscolorparserd.pdb CONFIGURATIONS "Debug" DESTINATION lib)
-        install(FILES $<TARGET_FILE_DIR:mbgl-vendor-nunicode>/mbgl-vendor-nunicoded.pdb CONFIGURATIONS "Debug" DESTINATION lib)
-        install(FILES $<TARGET_FILE_DIR:mbgl-vendor-parsedate>/mbgl-vendor-parsedated.pdb CONFIGURATIONS "Debug" DESTINATION lib)
+        install(FILES $<TARGET_FILE_DIR:mbgl-vendor-csscolorparser>/mbgl-vendor-csscolorparserd.pdb CONFIGURATIONS "Debug" DESTINATION "${CMAKE_INSTALL_LIBDIR}")
```

Comment:
IMHO these pdb install targets can be remoted totally. They should be automatic in CMake, dependin on the build type and compiler, no?